packages/components/src/Modal/GenericModal/subcomponents/ModalFooter/ModalFooter.tsx

```diff
@@ -26,44 +21,45 @@ export type ModalFooterProps = {
    * @deprecated Please use data-testid
    */
   automationId?: string
-  alignStart?: boolean
```

Comment:
So a quick squiz on metabase and it looks like this isn't use, which is why we're removing it yeah?

Reply:
Oh yeah it wasn't doing anything as well

Reply:
groovy, sounds good!